test.py

```diff
@@ -103,9 +103,10 @@ def test_bundle_from_cmd():
     min_redis_version = "4.6"
     min_redis_pack_version = "5.0"
     display_name = "test_module"
-    os_list = ["ubuntu14.04"]
+    module_name = "module_test"
+    os_list = ["ubuntu14.04","rhel7"]
 
-    argv = [MODULE_FILE_PATH, '-a', author, '-e', email, '-D', description, '-d', display_name, '-ol', os_list,
+    argv = [MODULE_FILE_PATH, '-a', author, '-e', email, '-D', description, '-d', display_name, '-n', module_name,
```

Comment:
Why did you omitted `os_list` ?

Reply:
Nice catch.  I added it back in and made sure the argument is different from the default